irohad/consensus/yac/transport/yac_pb_converters.hpp

```diff
@@ -24,8 +24,12 @@ namespace iroha {
           proto::Vote pb_vote;
 
           auto hash = pb_vote.mutable_hash();
-          hash->set_block(vote.hash.block_hash);
-          hash->set_proposal(vote.hash.proposal_hash);
+          auto hash_round = hash->mutable_vote_round();
```

Comment:
Maybe extract function for round (de-)serialization? So these functions will correspond to related proto types